src/library/browse/browsethread.cpp

```diff
@@ -154,17 +154,16 @@ void BrowseThread::populateModel() {
 
         const QString filepath = fileIt.next();
         {
-            TrackCacheLocker cacheLocker(
-                    TrackCache::instance().lookupOrCreateTemporaryForFile(
-                            filepath, thisPath.token()));
-            auto pTrack = cacheLocker.getTrack();
-            if (!pTrack) {
-                qWarning() << "Skipping inaccessible file"
-                        << filepath;
-                continue;
-            }
-
-            // Import metadata from file
+            TrackPointer pTrack =
+                TrackCache::instance().resolve(
```

Comment:
BaseTrackCache uses TrackCache this sounds like swapped class named from legacy.